site-packages/integralstor_utils/disks.py

```diff
@@ -217,6 +217,36 @@ def get_partitions(disk_name):
     else:
         return l, None
 
+def get_os_partition_stats():
+    ret_list = []
+    try:
+        lines, err = command.get_command_output('df -HT --exclude-type=devtmpfs --exclude-type=tmpfs --exclude-type=zfs')
+        if err:
+            raise Exception(err)
+        #print lines, err
+        num=0
+        ret_list = []
+        for line in lines:
+            if num == 0:
+                num += 1
```

Comment:
If the intention is to skip the first iteration of the sequence, then, will for line in lines[1:]: be a more idiomatic choice instead?